src/xdocs/writingchecks.xml

```diff
@@ -176,16 +178,15 @@ CLASS_DEF -> CLASS_DEF [5:0]
       </div>
       <p>
         As you can see very small java file transforms to a huge Abstract Syntax Tree, because
-        that is the most detailed tree including all components of the java file: object block,
+        this detailed tree includes all components of the java file: object blocks,
```

Comment:
`this detailed tree includes all components of the java file` => `this creates a very detailed tree and includes all the components of the java file`

Reply:
Changed to `this creates a very detailed tree that includes all the components of the java file`

Reply:
Ok.